docs/source/release_notes/release-notes.rst

```diff
@@ -50,6 +50,25 @@ develop
 .. <<<<------------------------------------------------------------------------------------------------------------->>>>
 
 ======
+0.62.2
+======
+
+27 October 2017
+
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *    - Type
+         - Change
+
+    *    - |fixed|
+         - Fixed the deprecated ``TransactionManagers.create`` methods, by specifying a default user agent if none was provided.
```

Comment:
Is it worth saying that they would previously have failed *at runtime*? The runtime-ness of those failures is rather sad

Reply:
yea that's an immutables thing. it's been a while since I've looked into it but have been told it's annoying to work around.